docs/gas_api/OpenApi.json

```
@@ -0,0 +1,253 @@
+{
+    "openapi":"3.0.2",
+    "info": {
+        "title":"GAS API",
+        "version":"1.0",
+        "description": "GAS API"
+    },
+    "tags": [
+        {
+            "name": "club"
+        }
+    ],
+    "paths": {
+        "?action=regist": {
+            "post": {
+                "tags": ["club"],
+                "summary": "創部申請",
+                "description": "部活動シートの末尾に申請のあった部活を追加します。",
+                "requestBody": {
+                    "required": true,
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "properties": {
+                                    "name": {
+                                        "type": "string"
+                                    },
+                                    "description": {
+                                        "type": "string"
+                                    },
+                                    "captain": {
+                                        "type": "string"
+                                    },
+                                    "collaborator1": {
```

```suggestion
                                    "collaborator_1 || collaborator_1st": {
```

細かいし宗教かもだけど、a-z.0-9 ではなく記号挟みたい
それと 1 だけでなく順番を連想されるように 1st とかにしたい

---

1st, 2ndの方で合わせました〜